TestJSEscaping fails locally

I'm adding this repo to Fedora to support updating Task to v3.38, but running into a failing test:

Local:
```
--- FAIL: TestJSEscaping (0.00s)
    exec_test.go:961: JS escaping [unprintable ﷿] got [unprintable ﷿] want [unprintable \uFDFF]
```

COPR (https://download.copr.fedorainfracloud.org/results/fuller/go-task/fedora-40-x86_64/07731787-golang-github-task-template/builder-live.log.gz):
```
--- FAIL: TestJSEscaping (0.00s)
    exec_test.go:961: JS escaping [unprintable ï·¿] got [unprintable ï·¿] want [unprintable \uFDFF]
```

This also fails when I simply clone the repo and run the test suite:
```
➜  template git:(main) go test .
--- FAIL: TestJSEscaping (0.00s)
    exec_test.go:961: JS escaping [unprintable ﷿] got [unprintable ﷿] want [unprintable \uFDFF]
FAIL
FAIL    github.com/go-task/template     0.072s
FAIL
```

--- a/exec_test.go
+++ b/exec_test.go
@@ -950,7 +950,7 @@ func TestJSEscaping(t *testing.T) {
 		{`'foo`, `\'foo`},
 		{`Go "jump" \`, `Go \"jump\" \\`},
 		{`Yukihiro says "今日は世界"`, `Yukihiro says \"今日は世界\"`},
-		{"unprintable \uFDFF", `unprintable \uFDFF`},
+		{"unprintable \uFFFE", `unprintable \uFFFE`},
 		{`<html>`, `\u003Chtml\u003E`},
 		{`no = in attributes`, `no \u003D in attributes`},
 		{`&#x27; does not become HTML entity`, `\u0026#x27; does not become HTML entity`},

